posts/tests.py

```
@@ -85,3 +92,101 @@ def test_controversial_content_sinks(self):
+
+        cls.community = Community.objects.create(
+            name="General", visibility="public", private=False, creator=cls.author
+        )
+
+        cls.post = Post.objects.create(
+            title="Old Legend",
+            author=cls.author,
+            community=cls.community,
+        )
+
+        cls.auth_headers = {"HTTP_AUTHORIZATION": f"Bearer some-random-jwt"}
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)